### PR TITLE
Add Notifications

### PR DESCRIPTION
**Pull request by aeddaqqa**
- add notification component
- add action to be fired when needed to activate the notification Snackbar
- customize the notification to have different types

### CODE REVIEW

File: src/components/Notification.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+type TransitionProps = Omit<SlideProps, 'direction'>
+
+function TransitionUp(props: TransitionProps) {
+    return <Slide {...props} direction="left" />
+}
+
+export default function Notifications() {
+    const notifStatus = useAppSelector(getNotificationStatus)
+    const notifMessage = useAppSelector(getNotificationMessage)
+    const notifSeverity = useAppSelector(getNotificationSeverity)
```
Comment (Chriserus) [inline src/components/Notification.tsx]: I, personally would just use full wording `notificationStatus` and such, but it's just a suggestion, not a requirement

**Review by Chriserus — DISMISSED** [review]

File: src/components/Notification.tsx (hunk 1 above)
Comment (Ayagoumi) [reply src/components/Notification.tsx]: @Chriserus  We usually use **get** or **set** to signify what's the purpose of the action, so I think this is a correct usage of the **get** notation.

File: src/views/wallet/WalletApp.tsx [hunk 2]
```diff
@@ -24,7 +33,8 @@ const LoadWallet = () => {
         setFetch(true)
     }
     useEffect(() => {
-        if (fetch) mount(ref.current, { setUpdateStore, setLogOut })
+        if (fetch)
+            mount(ref.current, { setUpdateStore, setLogOut, setAccount, dispatchNotification })
     }, [fetch])
```
Comment (Ayagoumi) [inline src/views/wallet/WalletApp.tsx:39]: Add an eslint-disable-line here

File: src/components/Notification.tsx (hunk 1 above)
Comment (Chriserus) [reply src/components/Notification.tsx]: I wasn't clear, I don't like `notifSeverity`, I would prefer `notificationSeverity`, same for other consts @Ayagoumi

File: src/views/wallet/WalletApp.tsx (hunk 2 above)
Comment (aeddaqqa) [reply src/views/wallet/WalletApp.tsx:39]: fixed

File: src/components/Notification.tsx (hunk 1 above)
Comment (Ayagoumi) [reply src/components/Notification.tsx]: @Chriserus this we agree upon, @aeddaqqa

Comment (aeddaqqa) [reply src/components/Notification.tsx]: fixed

**Review by Ayagoumi — DISMISSED** [review]
LGTM

**Review by Chriserus — DISMISSED** [review]

**Review by Chriserus — APPROVED** [review]

**Pull request merged**